README.md

```diff
@@ -11,7 +11,30 @@ Locally: This will run additional steps to first configure instances and access
 In CI: In this mode instance creation is completly skipped as instances will be provided to the role from Zuul node pool.
 
 To deploy locally, pass the local tag to do the initial deployment of instances that would be handled by Zuul's `nodepool`:
-`ansible-playbook infra-bootstrap.yml -e @my_vars.yml --tags local`
+`ansible-playbook infra-bootstrap.yml -e @my_vars.yml --tags **local**`
+
+The instances to deploy in the local cloud is defined in the `cifmw_bootstap_local_instances` variable. For example:
```

Comment:
Nit for next revision, should this be added to the `my_vars.yml` example below?

Reply:
Done.